### ISSUE TITLE
Data browser should update counts on refresh or collection selection

### ISSUE THREAD

**Issue by TylerBrock** [ParsePlatform/parse-dashboard#325]
Making this issue with the intention of closing it out myself.

When refreshing the data browser it would be great if the counts refreshed (both at the top of the page and on the sidebar).

I think I'd want similar behavior when selecting a different collection on the side bar (update of count) but I'm on the fence whether that is nice to have or not.

**Comment by durunvo** [comment]
Yes, that is kind of a little bit annoying me too. I need to refresh to web to get the number update.
As i understand, parse dashboard does't architech like that. It doesn't have a flux. I'm not sure if its good to have that.

**Comment by TylerBrock** [comment]
note: the sidebar should always reflect the count of all documents in the collection, the count at the top should reflect the filtered count

**Comment by grassland-curing-cfa** [comment]
The count of all documents is refreshed and are always correctly reflected on both the sidebar and at the top only if you run "Add a row" from the Edit menu. However, it is not refreshed if you try to add a new record from a REST call or via the JavaScript API, etc.

**Comment by grassland-curing-cfa** [comment]
Here is another weird thing!
I added a couple of new rows through REST calls. The count was not refreshed and reflected and so it remained "0". Then I refreshed the collection manually by clicking the Refresh button, then manually deleted those rows that were added. The count label then was refreshed and changed to a negative number on both the sidebar and the top.
![parsedashboard_negative](https://cloud.githubusercontent.com/assets/17244571/16830077/076aceda-49e0-11e6-8b9b-7471d1096f34.jpg)

**Comment by 185driver** [comment]
I see this issue is still present in dashboard version 1.0.19. Looking forward to a fix as time allows. Thanks.

**Comment by TylerBrock** [comment]
After some thought I believe I'd rather have (at least) the count of filtered objects shown. If nothing filtered than show the same count as the sidebar somewhere at the top.
